src/transformers/models/pegasus/configuration_pegasus.py

```diff
@@ -14,84 +14,48 @@
 # limitations under the License.
 """ PEGASUS model configuration """
 
+from ...configuration_utils import PretrainedConfig
 from ...utils import logging
-from ..bart.configuration_bart import BartConfig
 
 
 logger = logging.get_logger(__name__)
 
-# These config values do not vary between checkpoints
-DEFAULTS = dict(
-    vocab_size=96103,
-    max_position_embeddings=512,
-    d_model=1024,
-    encoder_ffn_dim=4096,
-    decoder_ffn_dim=4096,
-    encoder_attention_heads=16,
-    decoder_attention_heads=16,
-    encoder_layers=16,
-    decoder_layers=16,
-    dropout=0.1,
-    attention_dropout=0.1,
-    activation_dropout=0.1,
-    pad_token_id=0,
-    eos_token_id=1,
-    is_encoder_decoder=True,
-    normalize_before=True,
-    scale_embedding=True,
-    normalize_embedding=False,
-    add_final_layer_norm=True,
-    static_position_embeddings=True,
-    num_beams=8,
-    activation_function="relu",
-)
-# Config values that vary between checkpoints: for testing and conversion
-task_specific_params = {
```

Comment:
Are we sure no-one used that?

Reply:
We can move them to an example script but I agree they shouldn't be here.

Reply:
The only place where this was used was in a test that checks that the configs online like `google/pegasus-xsum` corresponds to the xsum task specific params there. I don't think this makes much sense...for me the configs should fully be self-explainable and don't need a hardcoded copy in transformers.